lib/ama_layout/navigation_helper.rb

```diff
@@ -0,0 +1,29 @@
+module AmaLayout
+  module NavigationHelper
+    def navigation
+      return AmaLayout::Navigation.non_member unless _member?(member?)
+      case
+      when _in_renewal_late?(status)
+        AmaLayout::Navigation.member_in_renewal_late
+      when _in_renewal?(in_renewal)
+        AmaLayout::Navigation.member_in_renewal
+      else
+        AmaLayout::Navigation.member
+      end
+    end
+
+    private
+
+    def _member?(member)
```

Comment:
@zoiec since these methods are always returning what is being passed to them, and that return value is essentially the value that you're passing in, could you drop the arguments on these three methods?

The only mod you'd have to make is line 18: member?